### PR TITLE
add ubiquerg recipe

### PR DESCRIPTION
**Pull request by michalstolarczyk**
Checklist

- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [X] Source is from official source
- [X] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [X] If static libraries are linked in, the license of the static library is packaged.
- [X] Build number is 0
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [X] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/ubiquerg/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
+{% set name = "ubiquerg" %}
+{% set version = "0.5.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "fb9f5cc40902d9327644471a7f5a69d2dcef36b3d4ea1485bc357ef4abdfcbaa"
```
Comment (marcelotrevisani) [inline recipes/ubiquerg/meta.yaml]:
```suggestion
  sha256: fb9f5cc40902d9327644471a7f5a69d2dcef36b3d4ea1485bc357ef4abdfcbaa
```

File: recipes/ubiquerg/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
```
Comment (marcelotrevisani) [inline recipes/ubiquerg/meta.yaml]:
```suggestion
  script: "{{ PYTHON }} -m pip install . -vv"
```

File: recipes/ubiquerg/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "fb9f5cc40902d9327644471a7f5a69d2dcef36b3d4ea1485bc357ef4abdfcbaa"
+
+build:
+  number: 0
```
Comment (marcelotrevisani) [inline recipes/ubiquerg/meta.yaml]:
```suggestion
  number: 0
  noarch: python
```

File: recipes/ubiquerg/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+  host:
+    - pip
+    - python
+  run:
+    - python
+
+test:
+  imports:
+    - ubiquerg
+  requires:
```
Comment (marcelotrevisani) [inline recipes/ubiquerg/meta.yaml]: if you need ``veracitools`` to run this project, ``veracitools`` should be in ``run``

File: recipes/ubiquerg/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+    - veracitools
+
+about:
+  home: "https://github.com/pepkit/ubiquerg/"
+  license: "BSD-2-Clause"
+  license_family: "BSD"
+  license_file: "LICENSE.txt"
+  summary: "Various utility functions"
+  doc_url: ""
+  dev_url: ""
```
Comment (marcelotrevisani) [inline recipes/ubiquerg/meta.yaml]:
```suggestion
```

File: recipes/ubiquerg/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+  requires:
+    - veracitools
+
+about:
+  home: "https://github.com/pepkit/ubiquerg/"
+  license: "BSD-2-Clause"
+  license_family: "BSD"
+  license_file: "LICENSE.txt"
+  summary: "Various utility functions"
+  doc_url: ""
```
Comment (marcelotrevisani) [inline recipes/ubiquerg/meta.yaml]:
```suggestion
```

File: recipes/ubiquerg/meta.yaml (hunk 4 above)
Comment (michalstolarczyk) [reply recipes/ubiquerg/meta.yaml]: is a package available here once the PR build is successful? Or it has to be merged first?

Comment (marcelotrevisani) [reply recipes/ubiquerg/meta.yaml]: It has to be merged, and it takes some time to get it build, packaged and uploaded to the anaconda cloud

**Comment by michalstolarczyk** [comment]
@marcelotrevisani  Any idea what's wrong with the appveyor build?

**Comment by marcelotrevisani** [comment]
> @marcelotrevisani Any idea what's wrong with the appveyor build?

I will push some modifications, this project needs the compilers right? So, it cannot be ``noarch``, I thought it was just a pure python project.

**Comment by marcelotrevisani** [comment]
> > @marcelotrevisani Any idea what's wrong with the appveyor build?
> 
> I will push some modifications, this project needs the compilers right? So, it cannot be `noarch`, I thought it was just a pure python project.

wait a minute... I think ``appveyor`` was building other recipes... I will investigate it a bit further

**Pull request merged**